admin_manual/installation/system_requirements.rst

```diff
@@ -23,7 +23,7 @@ For best performance, stability and functionality we have documented some recomm
 |                  | - openSUSE Leap 15.5                                                  |
 |                  | - CentOS Stream                                                       |
 +------------------+-----------------------------------------------------------------------+
-| Database         | - **MySQL 8.0+** or MariaDB 10.3/10.5/**10.6** (recommended)/10.11    |
+| Database         | - MySQL **8.0** / 8.4 or MariaDB 10.6/ **10.11** (recommended) / 11.4 |
```

Comment:
Let's recommend the latest LTS:
```suggestion
| Database         | - MySQL 8.0 / **8.4** or MariaDB 10.6/ **10.11** (recommended) / 11.4 |
```

Reply:
Reason I not did this: We have not yet have much experience with 8.4 as it is new to our test chain, but of course we can do so to recommend the latest LTS.
(8.0 has still ~2 years, 8.4 ~7 years)